api.lisp

```diff
@@ -68,6 +68,26 @@
         while (not (pipe-empty-p pipe))
         collect (funcall func (pipe-head pipe))))
 
+(defmacro map-node-sets->list (func &rest node-sets)
+  "@arg[func]{a function}
+   @arg[node-sets]{@class{node-set}}
+   @return{a list}
+   @short{Calls @code{func} for each tuple of nodes on the list @code{node-sets} and conses up
```

Comment:
This is a bit ambiguous. Maybe explicitly state that `func` is called with N arguments when `node-sets` consists of N node sets?

Reply:
Now reads: "Calls @code{func} with N arguments for each N-tuple of nodes
   on the N lists of @code{node-sets} and conses up a list of its
   return values"